R/DataMart.R

```diff
@@ -476,8 +476,8 @@ getApiRoot <- function(endpoint = "DataMart")
 {
     region <- URLencode(get0("region", ifnotfound = ""), TRUE)
     if (region == "") stopNotDisplayr("region")
-    if (region == "app")
-        region <- "displayr-app"  # to avoid Dynamic Site Acceleration CDN, which limits uploads to 100MB
+    if (region == "master")
+        region <- "displayr-master"  # to avoid Dynamic Site Acceleration CDN, which limits uploads to 100MB
```

Comment:
This will be good for tests in this PR but be bad for production right? The comment implies customers cant upload something larger than 100MB which might annoy someone.